### ISSUE TITLE
installed it, and then what???

### ISSUE THREAD

**Issue by PatrickMans** [kimai/kimai#255]
Having done all the steps in the installation guide, with no errors..
then what?? Where do I find the application? How can I use it? Do i have to start the app beforehand? 
Will this need the apache2 package to be installed or rather not?

**Comment by kevinpapst** [comment]
For testing, you can use the PHP built in webserver, described here:
https://github.com/kevinpapst/kimai2/blob/master/var/docs/installation.md#development-installation

For production use I would recommend nginx, but Apache works as well:
http://symfony.com/doc/current/cookbook/configuration/web_server_configuration.html

**Comment by PatrickMans** [comment]
I have configured the apache2 site to look at the public folder in /var/www/html/kimai2.
However, I had to change the .htaccess because it referred to app.php, which is not there. Changing it to the index.php finally gave me some output.

Thanks for the answer and the development of this tool!

**Comment by kevinpapst** [comment]
Ups, didn't even knew there was a .htaccess ... I am using nginx only.
I reopen this ticket, as this is a bug for some user.

**Comment by fedibelkacem** [comment]
I'm in a production environment. I don't own the server (not a dedicated server), I just have a hosting web solution (managing it under cPanel).

When I point my [sub]domain name to .../kimai2 directory, as there is no index.php, I got a webserver error. When I add /public/ I got some output as Patrick saying: No Look&Feel is applied!

When I pointed my [sub]domain name to .../kimai2/public directory I got the promised app :-)

By the way, I did not change any permission or ownership on kimai2 directory after cloning it. Default ones seem to work fine for the installation.
